openwisp_monitoring/monitoring/migrations/0012_rename_graph_chart.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 3.0.3 on 2020-06-01 17:04
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('monitoring', '0011_alert_settings_contenttype_registration'),
+    ]
+
+    operations = [
+        migrations.AlterModelTable(name='graph', table='monitoring_chart',),
+        migrations.RenameModel(old_name='Graph', new_name='Chart',),
+        migrations.AlterModelTable(name='chart', table=None,),
```

Comment:
@nepython I see you removed db_table but came back to this `migrations.AlterModelTable(name='chart', table=None,)`. Why is this necessary?

Reply:
I quote what I said on gitter last week or so:

> In any case, I don't see any problem if the table in the DB is not renamed, after all we are using an ORM that abstracts these details

We can squash all migrations before releasing the first version of openwisp-monitoring if necessary

Reply:
Last thoughts:

This issue was only about renaming the models. This PR contains also changes for the db schema, the css classes and some other extra changes. I'm not saying it's wrong but must stick to the issue description in general and open new issues and discuss if the scope gets broader, otherwise the changes are by default out of the scope of the issue.

Reply:
The renaming of other occurrences of graph is ok, I think it's consistent with the issue description.
I also don't understand very well why `table=None` is needed, @nepython can you please explain?

Reply:
I don't know for sure but AlterModelTable refers to the `db_table` in Meta class. Since, it is not declared, Django generates this migration but I have checked the database there were no changes actually. `table=None` means restore the default name of the model table which in this case happens to be the same as we have renamed two lines above. I tried without these as @PabloCastellano suggested no renaming at all but ended up losing data and there were some other issues cropping up when the table wasn't being renamed.